fastlane/metadata/fr-FR/subtitle.txt

```diff
@@ -1 +1 @@
-Construisez un site web ou un blog
\ No newline at end of file
+Construisez un site web
```

Comment:
Same note regarding the expected value and the upstream fix in GlotPress from the German version.